test/e2e/config/vsphere-ci.yaml

```diff
@@ -97,14 +97,7 @@ providers:
   - name: vsphere
     type: InfrastructureProvider
     versions:
-      - name: v0.8.2
-        value: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v0.8.2/infrastructure-components.yaml
-        type: "url"
-        contract: v1alpha4
-        files:
-          - sourcePath: "../../../test/e2e/data/infrastructure-vsphere/capi-upgrades/v1alpha4/cluster-template.yaml"
-          - sourcePath: "../../../metadata.yaml"
-      - name: v1.7.0
+      - name: v1.8.99
```

Comment:
I assume the idea is to get rid of multiple versions in this PR and then add them later when we add the clusterctl upgrade tests?

(Just wondering because we have a few more for core CAPI above)

Reply:
\+ I think this should be v1.9.99 now (same in other places of course)

Reply:
Yeah exactly - will add the other versions in a follow up when we need them - i.e. the upgrade tests.

Reply:
Will also review what we need in terms of CAPI in follow up PRs +/- a PR to clean up the vsphere-ci and vsphere-dev files in general

Reply:
Up to you if you want to drop the core CAPI v1alpha4 stuff. We'll probably never need it again